velero-2-mixin/README.md

```diff
@@ -0,0 +1,172 @@
+# Velero Mixin
+
+The Velero mixin is a set of configurable Grafana dashboards and alerts.
+
+The Velero mixin contains the following dashboards:
+
+- Velero cluster overview
+- Velero overview
+- Velero logs overview
+
+and the following alerts:
+
+- VeleroBackupFailure
+- VeleroHighBackupDuration
+- VeleroHighRestoreFailureRate
+- VeleroUpStatus
+
+## Velero Cluster Overview
+
+The Velero cluster overview dashboard a top level overview that provides details on alerts, volume snapshots, backup and restore counts.
+
+![Velero Overview Dashboard]()
+
+## Velero Overview
+
+The Velero overview dashboard provides details on backup sizes, backup success rates, backup status, restore counts, restore success rates, and snapshot data.
+
+![Velero Overview Dashboard]()
+
+## Velero Logs Overview
+
+The Velero logs overview dashboard provides details on the Velero system. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance.
```

Comment:
Double checking to see if this is needed when using k8s with the standard installation/helm chart.

Reply:
I will get rid of this part on the main branch which is where the README stuff lives.